kroxylicious-filters/kroxylicious-encryption/doc/design.adoc

```diff
@@ -0,0 +1,267 @@
+= Encryption Filter Design
```

Comment:
This `design.md` is substantially the same as in my draft PR for just the design doc. That one can be closed if this PR gets merged.